odbc/src/api/data.rs

```diff
@@ -1487,6 +1555,32 @@ mod unit {
                 test_it!(bson, v);
             });
         }
+        #[test]
+        fn string_conversions_to_floats() {
+            type V = Vec<(&'static str, f64, Result<(), ()>, Option<&'static str>)>;
+            let strings: HashMap<String, V> = map! {
+                (-PI).to_string() => vec![
+                    ("f64", -PI, Ok(()), None),
+                    ("f32", -PI, Ok(()), Some(FRACTIONAL_TRUNCATION)),
```

Comment:
I don't think it should (or will) return a Fractional truncation info. It looks like this was just overlooked and you should have None here, isn't it?
Same for PI string to f32 below.

Reply:
It definitely won't. It looks like I need to update the test harness; this should have failed!